packages/elements-dev-portal/src/hooks/useGetNodeContent.ts

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { useQuery } from 'react-query';
+
+import { PlatformUrlContext } from '../components/DevPortalProvider';
+import { getNodeContent } from '../handlers/getNodeContent';
+
+export function useGetNodeContent(
+  {
+    nodeSlug,
+    projectId,
+    branchSlug,
+  }: {
+    nodeSlug: string;
+    projectId: string;
+    branchSlug?: string;
+  },
+  requestHeaders?: Record<string, string>,
```

Comment:
I'm not sure I like this `requestHeaders` parameter. It's leaking the otherwise solid abstraction. Also as the rest are in a nicely structured object, this feels like something that was added last minute, feels like tech debt. What are you trying to achieve with it?

Reply:
Removed them in https://github.com/stoplightio/elements/pull/1191/commits/811e57e8e70e05040bee14a5d55287c6af516a67